config.ts

```diff
@@ -16,52 +16,52 @@
 
 import * as path from 'path';
 
-
 // Default configuration
 export interface Config {
   // Log levels: 0-disabled,1-error,2-warn,3-info,4-debug
   logLevel?: number;
 
+  // Whether to enable to Trace Agent or not.
+  // Once enabled, the Trace Agent may not be disabled.
   enabled?: boolean;
 
-  // If true, information about query parameters and results will be
-  // attached to spans representing database operations.
+  // If true, additional information about query parameters and results will be
+  // attached (as labels) to spans representing database operations.
   enhancedDatabaseReporting?: boolean;
 
-  // The maximum number of characters reported on a label value. This
+  // The maximum number of characters reported on a label value. This value
   // cannot exceed 16383, the maximum value accepted by the service.
   maximumLabelValueSize?: number;
 
   // A list of trace plugins to load. Each field's key in this object is the
   // name of the module to trace, and its value is the require-friendly path
-  // to the plugin.
-  // By default, all of the following plugins are loaded.
-  // Specifying a different object for this field in the configuration passed
-  // to the method that starts the trace agent will cause that object to be
-  // merged with this one.
+  // to the plugin. (See the default configuration below for examples.)
+  // Any user-provided value will be used to extend its default value.
   // To disable a plugin in this list, you may override its path with a falsy
   // value. Disabling any of the default plugins may cause unwanted behavior,
   // so use caution.
   plugins?: {
     [pluginName: string]: string;
   };
 
-  // The max number of frames to include on traces (0 disables)
+  // The max number of frames to include on traces; pass a value of 0 to
+  // disable stack frame limits.
   stackTraceLimit?: number;
 
-  // We buffer the captured traces for `flushDelaySeconds` before publishing
-  // to the trace API; unless the buffer fills up before then.
-  // See `bufferSize`.
+  // Buffer the captured traces for `flushDelaySeconds` seconds before
+  // publishing to the trace API, unless the buffer fills up first.
+  // Also see `bufferSize`.
   flushDelaySeconds?: number;
 
-  // If paths are present in this array, then these paths will be ignored before
-  // `samplingRate` based decisions are made. Paths must include a leading
-  // forward slash and be of the form:
-  //   /componentOne/componentTwo/...
-  // Paths can additionally be classified by regex in which case any path matching
-  // any provided regex will be ignored.
-  // We ignore the health checker probes (/_ah/health) by default.
-  ignoreUrls?: string[];
+  // URLs that partially match any regex in ignoreUrls will not be traced.
+  // In addition, URLs that are _exact matches_ of strings in ignoreUrls will
```

Comment:
Given the comment here: https://github.com/GoogleCloudPlatform/cloud-trace-nodejs/issues/574#issuecomment-337298135, you expect the exact match behaviorr to be removed in a semver major change in the future. If that's the case, it would be could to call this out as deprecated behavior so that users can avoid depending upon it.

Reply:
Good call, done